toolbox/AppModel.js

```diff
@@ -0,0 +1,143 @@
+            id: 'default',
+            useRoutes: true,
+            orientation: 'h',
+            children: this.createTabs()
+        });
+    }
+
+    //------------------------
+    // For override / extension
+    //------------------------
+    getTabRoutes() {
+        return [
+            {
+                name: 'home',
+                path: '/home'
+            },
+            {
+                name: 'components',
+                path: '/components',
+                forwardTo: 'default.components.leftRightChooser',
+                children: [
+                    {name: 'leftRightChooser', path: '/leftRightChooser'},
+                    {name: 'toolbar', path: '/toolbar'}
+                ]
+            },
+            {
+                name: 'containers',
+                path: '/containers',
+                forwardTo: 'default.containers.hbox',
+                children: [
+                    {name: 'hbox', path: '/hbox'},
+                    {name: 'vbox', path: '/vbox'},
+                    {name: 'resizable', path: '/resizable'},
+                    {name: 'tabPanel', path: '/tabPanel'}
+                ]
+            },
+            {
+                name: 'grids',
+                path: '/grids',
+                forwardTo: 'default.grids.standard',
+                children: [
+                    {name: 'standard', path: '/standard'},
+                    {name: 'grouped', path: '/grouped'},
+                    {name: 'dateColumn', path: '/dateColumn'},
+                    {name: 'boolCheck', path: '/boolCheck'}
+                ]
+            },
+            {
+                name: 'icons',
+                path: '/icons'
+            }
+        ];
+    }
+
+    createTabs() {
+        return [
+            {
+                id: 'home',
+                component: HomePanel
+            },
+            {
+                id: 'components',
+                orientation: 'v',
+                children: [
+                    {id: 'leftRightChooser', component: LeftRightChooserPanel},
+                    {id: 'toolbar', component: ToolbarPanel}
+                ]
+            },
+            {
+                id: 'containers',
+                orientation: 'v',
+                children: [
+                    {id: 'hbox', component: HboxContainerPanel},
+                    {id: 'vbox', component: VboxContainerPanel},
+                    {id: 'resizable', component: ResizableContainerPanel},
+                    {id: 'tabPanel', component: TabPanelContainerPanel}
+                ]
+            },
+            {
+                id: 'grids',
+                orientation: 'v',
+                children: [
+                    {id: 'standard', component: StandardGridPanel},
+                    {id: 'grouped', component: GroupedGridPanel},
+                    {id: 'dateColumn', component: DateColumnGridPanel},
+                    {id: 'boolCheck', component: BoolCheckGridPanel}
```

Comment:
We can use one grid - can just be the "primary" grid tab - to include any column renderers we wish to highlight, without needing a tab for each.